javageci-core/src/main/java/javax0/geci/annotationbuilder/AnnotationBuilder.java

```diff
@@ -0,0 +1,110 @@
+package javax0.geci.annotationbuilder;
+
+import javax0.geci.api.GeciException;
+import javax0.geci.api.Segment;
+import javax0.geci.api.Source;
+import javax0.geci.tools.AbstractJavaGenerator;
+import javax0.geci.tools.CompoundParams;
+import javax0.geci.tools.GeciReflectionTools;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.Set;
+
+import static javax0.geci.api.Source.Set.set;
+import static javax0.geci.tools.CaseTools.ucase;
+
+public class AnnotationBuilder extends AbstractJavaGenerator {
+
+    private static class Config {
+        private String in = "";
+    }
+
+    @Override
+    public void process(Source source, Class<?> klass, CompoundParams global) throws Exception {
+        try {
+            final var mnemonic = getMnemonic(klass);
+            final var keys = getImplementedKeysSorted(klass);
+            final var annotation = ucase(mnemonic);
+            final var in = config.in;
+
+            final var newSource = source.newSource(set(in), annotation + ".java");
```

Comment:
`final var newSource = source.newSource(set(in), annotation + ".java");`

to

`final var newSource = source.newSource(set(config.in), annotation + ".java");`

It is just as readable, and there is no need for the extra reference to the local variable `in`. When you read the code and see the local variable `in` you have to spend a few seconds reading the source thinking:

Why is there this local variable? What happens with it? Where does it change? Oh, it cannot change, it is `final`. Does the variable `config.in` changes and we should reserve the content for local use?

None of these. There is no reason for this local variable. This is just clutter.

Reply:
Understood. Removed.